util/src/builder/mod.rs

```diff
@@ -1,7 +1,9 @@
 //! Builders for large structs.
 #![allow(clippy::module_name_repetitions)]
 
+mod button;
 mod callback_data;
 pub mod command;
 
+pub use self::button::ButtonBuilder;
```

Comment:
Missing exports for error types.

```suggestion
pub use self::button::{ButtonBuilder, ButtonError, ButtonErrorType};
```